### PR TITLE
Instead of Unpack the Pack - use tags for update strategies

### PR DESCRIPTION
**Pull request by nusphere**
make a new major tag for a full Support of Doctrine Migration V3

### CODE REVIEW

**Comment by nusphere** [comment]
Lets tag the as `symfony/orm-pack: 2.0.0`

Reason:
the latest version of an pack should be the up-to-date version of the containing packages. A stick to V2 is the wrong way - until a unpack losing the idea of the dependencies. If someone add a new dependency to this pack - all unpacked versions of this pack will do not require this dependency. 

Instead of make the latest pack version sticky, serve 2 versions of this pack. a sticky and a updated one.

**Comment by nusphere** [comment]
I need to use `"symfony/orm-pack": "1.0.8 || ^2.0"` for now

**Comment by nusphere** [comment]
reference to #29

File: composer.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
     "description": "A pack for the Doctrine ORM",
     "require": {
         "composer/package-versions-deprecated": "*",
-        "doctrine/orm": "^2",
-        "doctrine/doctrine-bundle": "^2",
-        "doctrine/doctrine-migrations-bundle": "^2"
+        "doctrine/orm": "*",
+        "doctrine/doctrine-bundle": "*",
```
Comment (nicolas-grekas) [inline composer.json:9]: both these lines must be kept as is: when dbal/orm v3 will be released, adopting them should be opt-in

File: composer.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
...
+        "doctrine/doctrine-migrations-bundle": "*"
```
Comment (nicolas-grekas) [inline composer.json:10]: please use `^3`

**Review by nicolas-grekas — COMMENTED** [review]
This will reintroduce issues like #25 and #18. These cannot be just ignored.
Here are some suggestions to help move forward.

**Review by nicolas-grekas — APPROVED** [review]
OK to me as is once https://github.com/symfony/flex/pull/656 is merged + tagged.

To be tagged as v2.0.0

**Comment by nicolas-grekas** [comment]
Now that https://github.com/symfony/recipes/pull/797 is merged, the path is clear.

File: composer.json (hunk 2 above)
Comment (gggeek) [reply composer.json:10]: I agree with the above comment - why not make orm-pack 2.0 stick to migrations 3.x only ? It seems that we are chasing again the same problems we got when migrations-bundle got bumped from 2 to 3...

**Pull request merged**